index.html

```diff
@@ -35,23 +35,178 @@ <h1 class="primary-text">I’m Maroun Glad to see you!</h1>
-            </li>
-            <li>
-              <a href="#"><img src="icons/github.svg" alt="github" id="github"></a>
-            </li>
-            <li>
-              <a href="#"><img src="icons/v-sign.svg" alt="V-sign" id="V-sign"></a>
-            </li>
+            <li><a href="#"><img src="icons/twitter.svg" alt="twitter" id="twitter"></a></li>
+            <li><a href="#"><img src="icons/Linkedin.svg" alt="linkedin" id="linkedin"></a></li>
+            <li><a href="#"><img src="icons/medium.svg" alt="medium" id="medium"></a></li>
+            <li><a href="#"><img src="icons/github.svg" alt="github" id="github"></a></li>
+            <li><a href="#"><img src="icons/v-sign.svg" alt="V-sign" id="V-sign"></a></li>
           </ul>
       </section>
     </section>
-</body>
-</html>
\ No newline at end of file
+
+    <section class="works">
+      <div class="card">
+        <div class="card-structure">
+          <img src="images/image.png " class="img-displayer" alt="first_photo">
+
+          <h1 class="project-title">Tonic</h1>
+
+          <ul class="tags">
+            <li class="client">CANOPY</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">Back End Dev</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">2015</li>
+          </ul>
+
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-structure">
+          <img src="images/snap.png" class="img-displayer" alt="first_photo">
+
+          <h1 class="project-title">Multi-Post
+            Stories</h1>
+
+          <ul class="tags">
+            <li class="client">CANOPY</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">Back End Dev</li>
+            <li><img class="rectangle" src="images/Counter.png" alt="counter"></li>
+            <li class="role">2015</li>
+          </ul>
+
+          <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+          <ul class="tags2">
+            <li class="lang-block"><p class="lang-text">html</p></li>
+            <li class="lang-block"><p class="lang-text">css</p></li>
+            <li class="lang-block"><p class="lang-text">javaScript</p></li>
+          </ul>
+
+          <button type="button" class="card-button">See Project</button>
+        </div>
+      </div>
```

Comment:
fix the overlapping of the text as shown in the screenshot below

![wrong1](https://user-images.githubusercontent.com/58376020/121586083-88ab4280-ca33-11eb-9104-e8dc5b64df40.png)